### PR TITLE
fix: notebook wait page updates for API changes

### PR DESCRIPTION
**Pull request by stoksc**
## Description
Fix the notebook wait page by adding back in some old API fields.

## Test Plan
- [x] run a notebook, open it via the webui

## Commentary (optional)
I'm having a bad day.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: master/internal/task/allocation.go [hunk 1]
```diff
@@ -173,10 +173,18 @@ func (a *Allocation) Receive(ctx *actor.Context) error {
 		a.Cleanup(ctx)
 	case sproto.ContainerLog:
 		if a.req.StreamEvents != nil {
-			ctx.Tell(a.req.StreamEvents.To, sproto.Event{LogEvent: ptrs.StringPtr(msg.String())})
+			ctx.Tell(a.req.StreamEvents.To, sproto.Event{
```
Comment (stoksc) [inline master/internal/task/allocation.go:176]: For every struct initialization as reported by my editor, I just added the previously present fields.

File: master/internal/task/allocation.go [hunk 2]
```diff
@@ -173,10 +173,18 @@ func (a *Allocation) Receive(ctx *actor.Context) error {
...
+			})
 			if rc := a.req.LogBasedReady; rc != nil && !a.logBasedReadinessPassed {
 				if rc.Pattern.MatchString(msg.String()) {
-					ctx.Tell(a.req.StreamEvents.To, sproto.Event{ServiceReadyEvent: &msg})
+					ctx.Tell(a.req.StreamEvents.To, sproto.Event{
+						State:             a.state.String(),
+						IsReady:           a.logBasedReadinessPassed,
+						ServiceReadyEvent: &msg,
+					})
 					a.logBasedReadinessPassed = true
```
Comment (shiyuann) [inline master/internal/task/allocation.go:188]: Should this be set earlier?

Comment (stoksc) [reply master/internal/task/allocation.go:188]: genius

**Review by hkang1 — APPROVED** [review]
Webui portion lgtm! Thanks for the fix 🙏

**Pull request merged**